lektor/reporter.py

```diff
@@ -112,6 +122,7 @@ def build_artifact(self, artifact, build_func, is_current):
         try:
             yield
         finally:
+            report_artifact_built(artifact, is_current)
```

Comment:
This will be called from ANY reporter, but the function will only send messages to `ChangeStreamReporter`s

Reply:
Nice.  I think this does make more sense than having a specialized Reporter class to do this.